### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Made changes for landing page..
My first task on styling.. I am happy to correct and work on the feed back..
New repo - app-landing inside packages
Navigate file is for the grid layout and routing to different location.
Landing.ts is to show the component on side bar..

Have added changes suggested by Lillian which says when I resize the window and make it small, the component should appear one below the other.. Resize window should adjust the number of component displayed on the UI..

### CODE REVIEW

File: packages/apps/src/SideBar/index.tsx [hunk 1]
```diff
@@ -28,9 +28,10 @@ interface Props {
   isCollapsed: boolean;
   isMenuOpen: boolean;
   toggleMenu: () => void;
+  isAdvance: boolean;
 }
 
-function SideBarContainer ({ className, collapse, handleResize, isCollapsed, isMenuOpen, toggleMenu }: Props): React.ReactElement<Props> {
```
Comment (benxgao) [inline packages/apps/src/SideBar/index.tsx]: Can you plz revert the changes in this file, otherwise, it gonna have conflict with the SideBar PR?

File: packages/apps/src/Apps.tsx [hunk 2]
```diff
@@ -17,12 +17,15 @@ import { SideBarTransition, SIDEBAR_MENU_THRESHOLD } from './constants';
...
 import BN from 'bn.js';
+import routing from '@polkadot/apps-routing';
+import { useLocation } from 'react-router-dom';
 
 interface SidebarState {
   isCollapsed: boolean;
   isMenu: boolean;
   isMenuOpen: boolean;
   transition: SideBarTransition;
+  isAdvance: boolean;
```
Comment (benxgao) [inline packages/apps/src/Apps.tsx]: Can you plz give more details on this constant, and how would it affect the landing page?

Comment (KarishmaBothara) [reply packages/apps/src/Apps.tsx]: Yes.. I will add it.. This is when 
Screen Shot 2020-03-24 at 2 23 53 PM open advanced is clicked, we need to pass this information to have the advanced dropdown open..

File: packages/apps/src/SideBar/index.tsx (hunk 1 above)
Comment (KarishmaBothara) [reply packages/apps/src/SideBar/index.tsx]: I will let it be.. we can first merge your changes, I will resolve the conflict.

Comment (KarishmaBothara) [reply packages/apps/src/SideBar/index.tsx]: Rebased with ur changes..

File: packages/app-landing/src/Navigate.tsx [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    width: 477px;
+    color: #5c5d5d;
+    font-family: Lato;
+    font-size: 22px;
+    font-weight: bold;
+    letter-spacing: 0;
+    line-height: 32px;
+  }
+
+  .grid-container {
```
Comment (benxgao) [inline packages/app-landing/src/Navigate.tsx]: `styled-components` make us be able to make html elements(react components) and css styles be united together as a distributed component without being bridged with classname. classname is fine, but we have to define css classname naming convention structurally (like using BEM styles) and carefully, otherwise if we defined `grid-container` classname styles somewhere else in the app, its hard to track and workaround.

In the future, we could refactor styles with classname to be `styled-components` way so that we do not need to worry about classnaming convention, for example here could be refactored like:

```js
const GridContainer = styled.div`
  ...
`;
// Then refactor code in component html element from <div classname="grid-container">...<div/>
<GridContainer>
  ...
<GridContainer/>
```

File: packages/app-landing/src/Navigate.tsx [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+}
+
+export default styled(Navigate)`
+  padding-top: 90px;
+  height: 983px;
+  width: 100%;
+  background-color: #fafafa;
+
+  .explore-featured-fun {
+    margin-left: 79px;
```
Comment (benxgao) [inline packages/app-landing/src/Navigate.tsx]: We could always `px` as spacing unit, that is fine, as which was provided in invision by design team. But generally it is better to use `rem` (or `em` sometimes) to define spacing, normally `1rem == 16px`, which depends on the base font size of `html` tag in html (polkadot set html base font size as 14px, that is, in cennznet-ui currently 1rem == 14px). 

There are some blogs described the benefits of rem, and on my own, rem could ensure us define spacing sizes in a kind of rhythm and structure.

File: packages/app-landing/src/Navigate.tsx [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+    height: 58px;
+    width: 477px;
+    color: #5c5d5d;
```
Comment (benxgao) [inline packages/app-landing/src/Navigate.tsx]: We have scoped color defination in a file, and described in this PR - https://github.com/cennznet/apps/pull/17/files

So that whenever design team want to change color theming, we do not need to search `5c5d5d` across the app to replace with new color. And there is other benefits for color defination in file, for example you could aleways use `color: ${colors.red}`, `color: ${colors.brandPrimary}` or `color: ${colors.warning}` in your components without search the exact color code.

Would suggest refactor color implements in landing page with the color config.

**Comment by benxgao** [comment]
Nice job on the styles as your first styling art-craft~

Some suggestions on styling practices which could always be discussed, and no changes need to be refactored associated with the suggestions in this PR.

Except color definition refactoring, LGTM.

**Comment by KarishmaBothara** [comment]
> Nice job on the styles as your first styling art-craft~
> 
> Some suggestions on styling practices which could always be discussed, and no refactoring is needed associated with the suggestions in this PR. We can refactor it in the future.
> 
> Except color definition refactoring, LGTM.

As discussed I haven't made changes to the colouring part.. Reviewed it with Lillian.

File: packages/app-landing/package.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@polkadot/app-landing",
```
[the remark this answers is not in the document]
Comment (KarishmaBothara) [reply packages/app-landing/package.json]: sure..I will change it..
